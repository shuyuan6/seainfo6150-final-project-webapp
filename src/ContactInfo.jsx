import React from 'react'

const ContactInfo = () => {
    return (
        <div>
            <h1>Contact Us</h1>
            <p>
                <b>Email Address: </b><br /> 
                <span>medService@medications.com</span>
            </p>
            <p>
                <b>Address: </b> <br />
                <span>1234 S 9th Ave N</span> <br />
                <span>Seattle, WA 98123</span> 
            </p>

            <p>
                <b>Phone Number: </b> <br />
                <span>123-456-7890</span> <br />
            </p>

        </div>
    )
}

export default ContactInfo