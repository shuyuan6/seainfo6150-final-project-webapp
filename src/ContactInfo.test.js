import React from "react";
import { render } from "@testing-library/react";
import ContactInfo from "./ContactInfo.jsx";

it("renders the ContactInfo component correctly", () => {
  const { container } = render(<ContactInfo />);
  expect(container).toMatchSnapshot();
});
