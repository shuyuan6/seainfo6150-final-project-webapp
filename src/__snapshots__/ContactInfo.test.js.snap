// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ContactInfo component correctly 1`] = `
<div>
  <div
    class="container"
  >
    <h1>
      Contact Us
    </h1>
    <br />
    <p>
      <b>
          Email Address: 
      </b>
      <br />
      <br />
      <span>
          medService@medications.com
      </span>
    </p>
    <br />
    <p>
      <b>
          Address: 
      </b>
       
      <br />
      <br />
      <span>
          1234 S 9th Ave N
      </span>
       
      <br />
      <span>
          Seattle, WA 98123
      </span>
    </p>
    <br />
    <p>
      <b>
          Phone Number: 
      </b>
       
      <br />
      <br />
      <span>
          123-456-7890
      </span>
       
      <br />
    </p>
    <br />
  </div>
</div>
`;
